packages/react-native/template/README.md

```diff
@@ -0,0 +1,64 @@
+This is a new [**React Native**](https://reactnative.dev) project, bootstrapped using [`@react-native-community/cli`](https://github.com/react-native-community/cli).
+
+# Getting Started
+
+>**Note:** Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till "Creating a new application" step, before proceeding.
+
+## Step 1: Start the Metro Server
+
+First, you will need to start Metro, the JavaScript _bundler_ that ships _with_ React Native.
+
+To start Metro, run `yarn start` _inside your React Native project folder_:
+
+```bash
+yarn start
+```
+
+## Step 2: Start your Application
+
+Let Metro Bundler run in its _own_ terminal. Open a _new_ terminal _inside your React Native project folder_ and run the following:
+
+```bash
+# For Android
+yarn android
```

Comment:
Sorry for the churn, but could we revert this back to npx, like the current documentation, so that these instructions also work for npm users?

Reply:
I.e. let’s keep this in sync with what we decide on for the official documentation.

Reply:
> what we decide on for the official documentation.

@NickGerleman Pardon me but, can you point me to the comment where everyone agreed as I see only your comment on the website PR about the same? Just so that we don't waste the time and effort again...

@cortinico @kelset can I get "final"👍 to revert this, and switch back to `npx`?

>P.S.: TBH, with due respect; this "revert" would imply _something_ was wrong in this whole process.

Reply:
I think that Nick meant to say that we can reach a finalized decision in the other PR (https://github.com/facebook/react-native-website/pull/3732) and then come back to this. Let's do that.

Reply:
Sure :)